### DISCUSSION TITLE
Image classification | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1056] [Q&A]
# Image classification | Seeed Studio Wiki

Classify Dog and Cat from an image.

https://wiki.seeedstudio.com/tinyml_course_Image_classification_project/

**Comment by fjmsouza** [comment]
Hi, 
we need acquire RGB raw image, not RGB565,  to preprocess it before to run the inference. Is this possible?

**Comment by burzredion** [comment]
I got an error:
ERR: Camera is not initialized
Failed to capture image
why?

**Reply by bleaaach** [reply]
Hello, thank you for your feedback.
To help you resolve this issue more efficiently, we recommend that you contact our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io). Please include as much detailed information as possible in your email (such as the steps you followed, code configuration, and error logs).
Our technical support team will be able to provide you with faster and more detailed guidance.
